dags/contextual_services_import.py

```diff
@@ -2,8 +2,8 @@
 Runs a Docker image that imports Quicksuggest suggestions
 from Remote Settings to BigQuery.
 
-The container is defined in
-[docker-etl](https://github.com/mozilla/docker-etl/tree/main/jobs/quicksuggest2bq)
+See the [`quicksuggest2bq`](https://github.com/mozilla/docker-etl/tree/main/jobs/quicksuggest2bq)
+docker image defined in `docker-etl`.
```

Comment:
Likewise for https://workflow.telemetry.mozilla.org/tree?dag_id=contextual_services_import